<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>alterRoomForm</class>
 <widget class="QDialog" name="alterRoomForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alter Room Information</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>230</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel/Close</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="numberLabel">
        <property name="text">
         <string>Room Number: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="styleLabel">
        <property name="text">
         <string>Room Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="costLabel">
        <property name="text">
         <string>Room Cost:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="numberDisplay">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="styleDisplay">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="costDisplay">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Choose a room:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="roomNumsBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>80</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="newNumberLabel">
        <property name="text">
         <string>New Number:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="newStyleLabel">
        <property name="text">
         <string>New Style:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="newCostLabel">
        <property name="text">
         <string>New Cost:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLineEdit" name="newNumberLine">
        <property name="inputMask">
         <string>000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newStyleLine">
        <property name="inputMask">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="newCostLine">
        <property name="inputMask">
         <string>00.00</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>alterRoomForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>240</x>
     <y>242</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
